src/utils/src/backend-supports/projects-get-all.ts

```diff
@@ -42,4 +43,22 @@ const calculatePercentExpectedProgress = (start: Date, weeks: number, status: St
   }
 };
 
-export { projectDurationBuilder, calculateEndDate, calculatePercentExpectedProgress };
+const calculateTimelineStatus = (progress: number, expectedProgress: number): TimelineStatus => {
```

Comment:
Can we get a js doc comment here to better document the function?

Reply:
Documentation added.